state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -912,13 +913,13 @@ pub mod pallet {
 		pub fn get_scheduled_swap_legs(
 			swaps: Vec<Swap>,
 			base_asset: Asset,
-		) -> Result<Vec<SwapLegInfo>, ()> {
+			pool_sell_price: Option<SqrtPriceQ64F96>,
+		) -> Vec<SwapLegInfo> {
 			let mut swaps: Vec<_> = swaps.into_iter().map(SwapState::new).collect();
 
-			Self::swap_into_stable_taking_network_fee(&mut swaps)
-				.map_err(|_| log::error!("Failed to simulate swaps"))?;
+			let _res = Self::swap_into_stable_taking_network_fee(&mut swaps);
```

Comment:
I know have a comment further below, but might be worth leaving a comment here as well to explain why we can ignore the result.